servers/store/src/main/proto/table.proto

```diff
@@ -23,10 +23,8 @@ option java_generate_equals_and_hash = true;
 message Contents {
   oneof object_type {
     IcebergTable iceberg_table = 1;
-    HiveTable hive_table = 2;
```

Comment:
imho we shouldn't remove/reorder protobufs. Fine given its pre-1.0 but we shouldn't do so again